components/Header/Header.js

```diff
@@ -35,13 +45,16 @@ export default function Header() {
     pageTitle = "Create Your Pet";
   } else if (router.pathname === "/update") {
     pageTitle = "Update Your Pet";
+  } else if (router.pathname === "/graveyard") {
+    pageTitle = "Pet Cemetery";
+    isGraveyard = true;
   }
 
   if (!pageTitle) return null;
 
   return (
     <HeaderContainer>
-      <StyledHeader>{pageTitle}</StyledHeader>
+      <StyledHeader isGraveyard={isGraveyard}>{pageTitle}</StyledHeader>
```

Comment:
You should prefix props you pass to a styled component with "$", otherwise you will get an error message in your terminal:

```suggestion
      <StyledHeader $isGraveyard={isGraveyard}>{pageTitle}</StyledHeader>
```

And this element is not a header, but a heading, so you should rename it to `StyledHeading` or `Heading`.
After having done so, you can rename your `HeaderContainer` to `StyledHeader`.
By making this change, the names match what your components actually display.